.github/actions/generate-coverage/scripts/run_rust.py

```
@@ -136,6 +133,55 @@ def _uses_cranelift_backend(manifest_path: Path) -> bool:
+    bool
+        ``True`` if any ``[profile.*]`` section sets
+        ``codegen-backend = "cranelift"``; ``False`` otherwise.
+    """
+    try:
+        content = manifest_path.read_text(encoding="utf-8")
+    except (OSError, UnicodeDecodeError):
+        return False
+    in_profile_section = False
+    for line in content.splitlines():
+        stripped = line.strip()
+        if not stripped or stripped.startswith("#") or stripped.startswith("//"):
```

Ruff (0.15.10)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)